### PR TITLE
Bump mytoyota version to 2.1.4

### LINKED ISSUE #286: stopped working completely on 8 August
**Describe the bug**
Upgraded HA last night to 2024.8.0 and noticed this morning that all the entities were not available.  Waited a few hours, still unavailable.  Deleted the car, restarted HA and then added back my username and password.  1 entry successful, but the car (device) and entities didn't show up, again did a restart and reload a few times.
Screenshot 2024-08-08 at 15 27 49

### PR DESCRIPTION
**Pull request by ajain-93**
Fixes #286

### CODE REVIEW

**Comment by ajain-93** [comment]
@DurgNomis-drol Can you review this PR to close #286?

File: custom_components/toyota/manifest.json [hunk 1]
```diff
@@ -7,6 +7,6 @@
   "documentation": "https://github.com/DurgNomis-drol/ha_toyota",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/DurgNomis-drol/ha_toyota/issues",
-  "requirements": ["mytoyota==2.1.1", "arrow"],
```
Comment (CM000n) [inline custom_components/toyota/manifest.json:10]: Maybe we shouldn't pin the versions here anymore, but simply allow all non-major updates?
So something like `"mytoyota>=2.1.1,<3.0.0"`

**Comment by desosav** [comment]
Just letting you know that there is already a PR that takes care of the manifest file https://github.com/DurgNomis-drol/ha_toyota/pull/257/files
Thank you for updating the mytoyota python module and bringing this integration back to life :)

**Review by CM000n — CHANGES_REQUESTED** [review]

**Comment by CM000n** [comment]
> Just letting you know that there is already a PR that takes care of the manifest file https://github.com/DurgNomis-drol/ha_toyota/pull/257/files Thank you for updating the mytoyota python module and bringing this integration back to life :)

Oh, that's right. There's already an MR from me :D I'm therefore closing this one as a duplicate.

**Comment by CM000n** [comment]
Dupliacte

**Pull request closed without merging**